src/EUTelProcessorAnalysisPALPIDEfs.cc

```diff
@@ -762,7 +781,7 @@ void EUTelProcessorAnalysisPALPIDEfs::processEvent(LCEvent *evt)
                     nResidualXPixel2by2[chi2Max[i]][index]->Fill(fmod(xposfit,2*xPitch),fmod(yposfit,2*yPitch));
                     nResidualYPixel2by2[chi2Max[i]][index]->Fill(fmod(xposfit,2*xPitch),fmod(yposfit,2*yPitch));
                   }
-                  else if (chi2 < chi2Max[i] && (yposfit < 9.2 || yposfit > 12.7 || xposfit > 27.4 || xposfit < 1.2))
+                  else if (chi2 < chi2Max[i] && (yposfit < 9.2 || yposfit > 12.7 || xposfit > 27.4+1.3 || xposfit < 1.2 - 0.2))
```

Comment:
I think these numbers, including the numbers to cut on the hole of the tracking planes should be parameters read from the steering templates.